web/index.html

```diff
@@ -376,32 +437,36 @@ <h3>Used in</h3>
             return queryParams;
         }
 
-        // Decode base64 value mod 1000
-        function getChunk(value) {
-            let base64 = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-='
-            let result = 0
-            for (let i = 0; i < value.length; i++) {
-                result = (result * 64 + base64.indexOf(value[i]))
-            }
-            return Math.floor(result / 100)
-        }
-
-        const DATA_PREFIX = 'data/';
+        const PATH_PREFIX = window.location.host == 'expitau.github.io' ? '/' : '';
+        const CHUNK_WORKER_PATH = `${PATH_PREFIX}loadChunkWorker.js`;
+        const DATA_PATH = `${PATH_PREFIX}data/`;
+        const HELPER_PATH = window.location.host == 'expitau.github.io' ? 'https://expitau.github.io/InfiniteCraftWiki/helper'
+            : 'http://127.0.0.1:5501/web/helper.html';
 
         // Vue app
         const app = new Vue({
             el: '#app',
             data: {
                 data: { index: {}, list: [], costs: {}, craftingCache: {}, data: {}, metadata: { recipeCount: 0, loadedChunks: [] } },
+                chunkLoadState: [],
```

Comment:
This is used by the lazy loading to hold all of the current chunk load states. If a chunk is being loaded it is added to this array as a Promise. If another function tries loading a chunk that already exists in this array it will be returned the element in this array for that chunk. This means that already loaded chunks will not be reloaded, and if a chunk is requested that is currently loading, then even if that chunk load was not called by the current function, it is still able to await the result of that chunk load, which is incredibly helpful.

Reply:
oh yeah, that's fair lol. Fixed that.